src/formats/PDB.cpp

```diff
@@ -249,62 +273,74 @@ void PDBFormat::read_HELIX(string_view line) {
         return;
     }
 
-    switch (helix_type) {
-    case 1: // Treat right and left handed helixes the same.
-    case 6:
-        secinfo_.insert({ start_info, std::make_pair(end_info, "alpha helix") });
-        break;
-    case 2:
-    case 7:
-        secinfo_.insert({ start_info, std::make_pair(end_info, "omega helix") });
-        break;
-    case 3:
-        secinfo_.insert({ start_info, std::make_pair(end_info, "pi helix") });
-        break;
-    case 4:
-    case 8:
-        secinfo_.insert({ start_info, std::make_pair(end_info, "gamma helix") });
-        break;
-    case 5:
-        secinfo_.insert({ start_info, std::make_pair(end_info, "3-10 helix") });
-        break;
-    default:
-        break;
+    // Convert the code as a character to its numeric meaning.
+    // See http://www.wwpdb.org/documentation/file-format-content/format23/sect5.html
+    // for definitions of these numbers
+    std::string type;
+    if (helix_type == 1) {
```

Comment:
This makes me miss Rust.

Reply:
Would a vector of strings be better here? This would effectively 'map' the integer to the decoded string and may improve coverage.

Reply:
That's a good idea!